conf/messages.en

```diff
@@ -1606,6 +1607,7 @@ rRAllowancesStart.bullet.content.l3 = replacing domestic goods
 rRAllowancesStart.bullet.content.l4 = other capital allowances
 rRAllowancesStart.details.content.l5 = Capital allowances for a car
 
+# Remove?
```

Comment:
Forgotten or on purpose? Thanks

Reply:
On purpose: Just a marker to remove those messages, it' not used anymore.